src/SFA.DAS.ProviderApprenticeshipsService.Web/Helpers/CsvWriterFacade.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.IO;
+using System.Web.Mvc;
+using CsvHelper;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Web.Helpers
+{
+    public static class CsvWriterFacade
```

Comment:
Does this facade have to be static. Could we not instantiate the facade, and compose it of the CsvWriter? For the Excel implementation, we instantiate the workbook and Deliver() it.